.github/workflows/ci.yml

```diff
@@ -32,7 +32,12 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
       - name: Build with Maven
-        run: mvn clean install
+        run: |
+          mvn clean install | tee ./mvn.log
+          if [[ $(grep -c "overlapping classes" ./mvn.log) -gt 0 ]];then
```

Comment:
It takes me some time to figure a "bug" of github actions. The `grep` just does not work as expected.